apps/web/components/RecommendedProducts/RecommendedProducts.vue

```diff
@@ -1,9 +1,12 @@
 <template>
+  <p v-if="headline" data-testid="recommended-products" class="mb-4 typography-text-lg text-center md:text-left">
```

Comment:
```suggestion
  <p v-if="headline" data-testid="recommended-products-headline" class="mb-4 typography-text-lg text-center md:text-left">
```
I would be more specific here. `recommended-products` reads like it would be for the whole component and not just one part of it.